src/ui/public/filter_editor/filter_editor.less

```diff
@@ -0,0 +1,27 @@
+@import (reference) "~ui/styles/mixins";
+
+.filterEditor {
+  position: absolute;
+  width: 600px;
+  z-index: 101;
+}
+
+.ui-select-choices--autoWidth {
```

Comment:
That looks a lot better :+1: I'm wondering if it would be worth breaking with the ui-select naming scheme and go with our camel-case scheme (`uiSelectChoice`) or if it's better to break with our convention. @cjcenizal, any opinion?